docs/data/material/components/masonry/MasonryWithVariableHeightItems.js

```diff
@@ -27,7 +26,7 @@ export default function MasonryWithVariableHeightItems() {
           <Paper key={index}>
             <StyledAccordion sx={{ minHeight: height }}>
               <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-                <Typography>Accordion {index + 1}</Typography>
+                <span style={{ textAlign: 'left' }}>Accordion {index + 1}</span>
```

Comment:
Why is `textAlign: left` needed?

Reply:
The text alignment was left-aligned previously because `Typography` had that as its default, preventing Argos changes. 

### Left-aligned text: 

![image](https://github.com/user-attachments/assets/d098a6ca-4e13-4d69-b940-29a34fc82323)  

### Without left alignment:  

![Screenshot (30)](https://github.com/user-attachments/assets/134f1e76-79fa-498c-b04f-d0c1f3f45426)  

It's probably fine to leave it without left alignment.

Reply:
What is causing the change in the alignment? The removal of `<Typography>` from the demos? If that's the case, I wonder if `AccordionSummary` should align text to the left (start) by default.

Reply:
> What is causing the change in the alignment? The removal of <Typography> from the demos?

Yes, I also noticed this in the Accordion demos during screen resize. The issue is due to changing the summary content from a block element (`div`) to an inline element (`span`). The block-level text aligns to the left for LTR languages and inline elements like `span` do not control text alignment themselves. Adding `text-align: start` to the `<span>` fixed it. Nice catch!